src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
```

Comment:
Move resetting margins and paddings to specific tags, and better to external reset.css file